rest/model/task.go

```diff
@@ -74,6 +74,8 @@ type APITask struct {
 	CanSync                 bool                `json:"can_sync,omitempty"`
 	SyncAtEndOpts           APISyncAtEndOptions `json:"sync_at_end_opts"`
 	Ami                     *string             `json:"ami"`
+	HasCedarResults         bool                `json:"has_cedar_results"`
+	CedarResultsFailed      bool                `json:"cedar_results_failed"`
```

Comment:
nit. I'm assuming this is only used because we want to access this field from a child resolver without fetching the db task.
 https://github.com/evergreen-ci/evergreen/pull/5066/files#diff-a3ce50f20b746b196f29c6d414e685b0110c24b768bcf83756de8d842c98f6dcR2703 

Would we be able to specify `json:"-" for these fields` so they doesn't show up in an api response. We had previously spoken to the app team about reducing the amount of fields we add to the apiModels.

Reply:
Yeah, or we can add an `omitempty` so it only appears if there are cedar test results or if the cedar test results failed. Would it ever be useful outside of the resolvers to use these fields?

Reply:
I don't think we wan't/need to expose these fields to users. Since we are only using them internally to check if the values are populated to support some other functionality we don't ever return them or expose them to the user.

Reply:
SGTM, I will put up another PR to set the json tags accordingly.